lib/swaps/Swaps.ts

```diff
@@ -174,7 +174,10 @@ class Swaps extends EventEmitter {
   /**
    * Sends an error to peer. Sets reqId if packet is a response to a request.
    */
-  private sendErrorToPeer = async (peer: Peer, rHash: string, failureReason: SwapFailureReason, errorMessage?: string, reqId?: string) => {
+  private sendErrorToPeer = async ({
+    peer, rHash, failureReason = SwapFailureReason.UnknownError, errorMessage, reqId }:
+    {peer: Peer, rHash: string, failureReason?: SwapFailureReason, errorMessage?: string, reqId?: string,
```

Comment:
These lines are a little bit hard to read, could you put the open and close `{}` on the same line? It looks off to me when one of the two braces is in-line with the properties but the other is not.